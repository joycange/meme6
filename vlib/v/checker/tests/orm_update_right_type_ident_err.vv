module main

import db.sqlite

pub struct SeoBlock {
    pub:
    seo_f1 string
    seo_is_f1_capitalize bool
    seo_params string
}

fn main() {
	db := sqlite.connect('test.db') or { panic(err) }

    sql db {
        update SeoBlock set seo_f1 = seo_f1 where seo_f1 == ""
    } or { panic(err) }
}
