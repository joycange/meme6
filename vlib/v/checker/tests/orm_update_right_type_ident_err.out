vlib/v/checker/tests/orm_update_right_type_ident_err.vv:16:38: error: unknown V identifier `seo_f1`
   14 | 
   15 |     sql db {
   16 |         update SeoBlock set seo_f1 = seo_f1 where seo_f1 == ""
      |                                      ~~~~~~
   17 |     } or { panic(err) }
   18 | }
